database/migration/migrateData.ts

```diff
@@ -1088,6 +1088,36 @@ const migrateData = async () => {
     `)
   }
 
+  // v1.0.0
+  if (databaseVersionLessThan('1.0.0')) {
+    console.log('Migrating to v1.0.0...')
+
+    console.log(' - Adding reviewer/assigner lists to applications')
+    await DB.changeSchema(`
+      ALTER TABLE public.application   
+        ADD COLUMN IF NOT EXISTS reviewer_list VARCHAR[],
+        ADD COLUMN IF NOT EXISTS assigner_list VARCHAR[];
+    `)
+
+    console.log(' - Adding application_reviewer_action table')
+    await DB.changeSchema(`
+      CREATE TABLE IF NOT EXISTS public.application_reviewer_action (
+        id serial PRIMARY KEY,
+        user_id integer REFERENCES public.user(id)
```

Comment:
I think we need to index user_id and application_id (indexing actions might also help)

Reply:
Yeah good call: 9bd2a3c1

We never query the table by the actions, so probably wouldn't help to index those columns at this point.